src/style.css

```diff
@@ -0,0 +1,124 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
Checklist!
You can use `*` selector to reset the margin of all elements on the page!
check this link [* Selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Universal_selectors)